src/main/java/com/cdancy/bitbucket/rest/BitbucketApi.java

```diff
@@ -20,23 +20,24 @@
 import java.io.Closeable;
 
 import com.cdancy.bitbucket.rest.features.AdminApi;
-import com.cdancy.bitbucket.rest.features.BuildStatusApi;
 import com.cdancy.bitbucket.rest.features.BranchApi;
+import com.cdancy.bitbucket.rest.features.BuildStatusApi;
 import com.cdancy.bitbucket.rest.features.CommentsApi;
 import com.cdancy.bitbucket.rest.features.CommitsApi;
 import com.cdancy.bitbucket.rest.features.DefaultReviewersApi;
 import com.cdancy.bitbucket.rest.features.FileApi;
 import com.cdancy.bitbucket.rest.features.HookApi;
+import com.cdancy.bitbucket.rest.features.InsightsApi;
 import com.cdancy.bitbucket.rest.features.ProjectApi;
 import com.cdancy.bitbucket.rest.features.PullRequestApi;
 import com.cdancy.bitbucket.rest.features.RepositoryApi;
+import com.cdancy.bitbucket.rest.features.SearchApi;
 import com.cdancy.bitbucket.rest.features.SyncApi;
 import com.cdancy.bitbucket.rest.features.SystemApi;
+import com.cdancy.bitbucket.rest.features.KeysApi;
```

Comment:
:warning: Wrong lexicographical order for 'com.cdancy.bitbucket.rest.features.KeysApi' import. Should be before 'java.io.Closeable'. (#QR83:37#)-CHECKSTYLE_CustomImportOrderCheck